### PR TITLE
Adding instruction + template for facilitator prep before story session

### PR DESCRIPTION
**Pull request by amcasari**
1. Added in a checklists w/ notes on prepping for facilitating a session
2. Added a prep "session flow" doc w/ spaces for personal information about the storyteller + bio notes

### CODE REVIEW

File: facilitator-prep.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+* Life happens and schedules change. Communicate as early as possible with your storyteller and our project leads if you need to reschedule or cancel a session.
+
+## 2. Do your facilitation homework.
+
+* Create a copy of [Storyteller Session Prep](https://github.com/opensourcestories/storytelling/session-prep-template.md).
+
+  * Coordinate with storyteller to fill out personal information in the prep material.
+
+  * Choose your Flow from [Conversation Flow for Open Source Storytellers](bit.ly/opensource-storytelling-conversation-flow)
```
Comment (juliaferraioli) [inline facilitator-prep.md:19]: We should probably update this doc to reflect that sessions are now half the time.

File: facilitator-prep.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  * Additional sample questions are available in our shared [story-questions bank](https://github.com/opensourcestories/story-questions).
+
+* We respect the time we spend with each other. Read the prep doc.
+
+## 3. Check your recording and admin setup.
+
+* Are you signed into the communication channel you've been using to coordinate with the storyteller?
+
+* Have you checked your audio recording channels?
```
Comment (juliaferraioli) [inline facilitator-prep.md:29]:
```suggestion
* Have you checked your audio recording channels? We recommend (if possible) capturing each audio stream separately.
```

File: session-prep-template.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# [INSERT STORYTELLER NAME] Prep
+
+## Personal information
```
Comment (juliaferraioli) [inline session-prep-template.md:3]: This is all captured in the Calendly invitation, so we don't need the facilitator to re-ask the storyteller to provide this information.

File: session-prep-template.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+* Choose your Flow from [Conversation Flow for Open Source Storytellers](bit.ly/opensource-storytelling-conversation-flow)
+
+* Additional sample questions are available in our shared [story-questions bank](https://github.com/opensourcestories/story-questions)
+
+* Remember to ask for parting thoughts + thank the storyteller
+
+### PRODUCTION NOTE FOR STORYCORPS CONNECT
+
+* DO NOT CLICK STOP UNTIL YOU ARE DONE
```
Comment (juliaferraioli) [inline session-prep-template.md:27]: 😆

File: session-prep-template.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## After recording notes to share with storyteller
+
+* Transcription make take a couple of weeks, after that we will share the draft with you
+* If we don’t get any corrections within ~4 days (though that’s negotiable if something comes up) we will go ahead and publish it
+* You can always request changes to the published version if you catch something later!
+
+
```
Comment (juliaferraioli) [inline session-prep-template.md:35]: Extraneous blank line

**Review by juliaferraioli — CHANGES_REQUESTED** [review]
Some suggestions.

File: facilitator-prep.md (hunk 1 above)
Comment (amcasari) [reply facilitator-prep.md:19]: Updated w/ 
* new question ratio suggestions (15-mins + 30-mins)
* split out the intro + warm-up question
* started a section for warm-up question
* created [a new issue in story-questions]((https://github.com/opensourcestories/story-questions/issues/3)) to remind myself we need that bank

File: session-prep-template.md (hunk 3 above)
Comment (amcasari) [reply session-prep-template.md:3]: Yes. I was moving our Google Doc template that we use to prep before a session into a template here. 

Should the change here be an additional annotation in the template as well as a change in wording to the facilitator checklist?

Comment (juliaferraioli) [reply session-prep-template.md:3]: I think the updates you made resolve my concern.

File: session-prep-template.md (hunk 5 above)
Comment (amcasari) [reply session-prep-template.md]: 🤦‍♀️

**Comment by amcasari** [comment]
Thanks @juliaferraioli! I'm going to give @Ruth-ikegah a chance to check this PR against our recent conversation. Ruth, no rush.

**Review by juliaferraioli — APPROVED** [review]
LGTM but waiting for @Ruth-ikegah to review before merging.

**Review by Ruth-ikegah — APPROVED** [review]
LGTM too! Sorry for the delay!

**Pull request merged**